src/Storage/Storage.Test/app.config

```diff
@@ -1,11 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why remove this? Please do not change artifacts outside your module and test folders

Reply:
I think it was deleted unknowningly

Reply:
.gitignore suggest the following . Please advise next steps on this
# app.config is not useful for PowerShell.
# please do not commit any app.config files.
app.config